### DISCUSSION TITLE
Any chance for WingetUI Microsoft store version?

### DISCUSSION THREAD

**Discussion by itayking1990** [Devolutions/UniGetUI#1991] [General]
I would like to inquire about the possibility of publishing WingetUI in the Microsoft Store. Is it technically feasible, or would the updated mechanics make it impossible?

**Comment by marticliment** [comment]
I tried, but it doesn't pass certification since it installs apps from. third party sources

**Reply by itayking1990** [reply]
Maybe you could email Microsoft to see if there is a workaround or maybe you already tried such a shame Microsoft not allow it.

**Reply by marticliment** [reply]
I have tried a few times, and I am going to try again argumenting that wingetui does not directly install apps, only uses package managers, but they will probably laugh at my face.

while ms store may be less likely to be loved by users, their security and reliability standards are very high, and WingetUI cannot achieve such levels
